### ISSUE TITLE
Some questions about model training

### ISSUE THREAD

**Issue by sunhan1997** [NVlabs/FoundationPose#15]
Thank you so much for your excellent work!  I have run this code successfully, and the demo is amazing. I have some questions about the model.  In the pose estimation section, the translation appears to be obtained from the depth map firstly, then the 252 template rotations are rendered, and then pose refinement and pose selection are performed. In the pose track, the pose refinement  module is continuously running. It is obvious that the pose refinement  is very important in this job.

 It seem to be that the pose refinement  module  utilizes a low-parameter network that iteratively predicts residuals rotation and  translation. I wonder  why such a low-parameter network can be effectively trained on large-scale datasets, and obey scaling law.  

Additionally, compared with diff-dope (https://github.com/NVlabs/diff-dope), the pose refinement  module in this paper is faster and more accurate. Does this mean that refineing pose directly using differentiable rendering is not an efficient method and that it is better to predict residual pose with a network?

Looking forward to your reply very much. Thank you again for your work.

**Comment by wenbowen123** [comment]
- the pose refinement only needs to learn local adjustment. This simplifies learning task.
- The transformer + large scale training probably gives birth to the emergent behavior of great generalization to many unseen objects
- Diff-dope needs hyperparameter tuning of the learning rate and choices of the modalities. It's not learning-based but more like a per-scene optimization, and thus it's also slower with that online optimization, while the pose refiner in this work is purely inference.

**Comment by sunhan1997** [comment]
> * the pose refinement only needs to learn local adjustment. This simplifies learning task.
> * The transformer + large scale training probably gives birth to the emergent behavior of great generalization to many unseen objects
> * Diff-dope needs hyperparameter tuning of the learning rate and choices of the modalities, while pose refiner in this work is all learned priorly and generalizes well with the same configuration.
> * Diff-dope is not learning-based but more like a per-scene optimization, and thus it's also slower with that online optimization, while the pose refiner in this work is purely inference.

Thank you very much for your patient response.  In the ablation study,  the transformer-based architecture is replaced by convolutional and linear layers, but it still achieve impressive performance. It seems that the CNN model can also be trained on large-scale dataset and get great generalization?

**Comment by wenbowen123** [comment]
yes, CNN still has its place, though not as well-performing as transformers in this case.
